src/Native/build-native.sh

```diff
@@ -2,11 +2,13 @@
 
 usage()
 {
-    echo "Our parameters changed! The parameters: buildArch, buildType and HostOS"
-    echo "are passes by the Run Command Tool."
+    echo "Our parameters changed! The parameters: buildArch, buildType, buildOS, numProc"
+    echo "are passed by the Run Command Tool."
+    echo "If you plan to only run this script, be sure to pass those parameters in the stated order."
```

Comment:
Here's an example:

https://github.com/dotnet/core-setup/blob/master/src/corehost/build.sh#L67

All you need to do is call `shift` and then read the next parameter ($1)